pallets/runtime/tests/src/asset_test.rs

```diff
@@ -2922,3 +2922,39 @@ fn schedule_remaining_works() {
         assert_ts(5);
     });
 }
+
+#[test]
+fn mesh_1531_ts_collission_regression_test() {
+    ExtBuilder::default().build().execute_with(|| {
+        // Create the assets.
+        let owner = User::new(AccountKeyring::Alice);
+        let asset = |name: &[u8]| {
```

Comment:
We could use `pallet/common/src/benchs/asset.rs : make_asset` here.

Reply:
We decided to do this later, tracked in https://polymath.atlassian.net/browse/MESH-1532.